apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/shard/helper/csv/ValueParserTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c)  2018-2019. Apollo Foundation.
+ */
+
+package com.apollocurrency.aplwallet.apl.core.shard.helper.csv;
+
+import com.apollocurrency.aplwallet.apl.core.shard.helper.ValueParserImpl;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ValueParserTest {
+    private ValueParser parser = new ValueParserImpl();
+
+    @Test
+    void parseQuotedStringObject() {
+        String o = "'APL-ZW95-E7B5-MVVP-CCBDT'";
+        String expected = o.substring(1, o.length()-1);
+        assertEquals(expected, parser.parseStringObject(o));
+    }
+
+    @Test
+    void parseStringObject() {
+        String o = "APL-ZW95-E7B5-MVVP-CCBDT";
+        assertEquals(o, parser.parseStringObject(o));
+    }
```

Comment:
please, consider JUnit 5 Parameterized Tests to combine these 2 test cases into a single one

Reply:
done